tests/kvasir-tests/Makefile.common

```diff
@@ -121,6 +121,9 @@ endif
 endif
 
 ifeq ($(OS_ID),Ubuntu)
+ifeq ($(OS_REL),23.10)
+	GOAL := goal-Ubuntu2310
```

Comment:
Since Ubuntu 24.04 has been released (and it's LTS), should we just ensure that we never run tests under anything between 22.04 (the last LTS Ubuntu release) and 24.04?  That would simplify the logic here.  I would need to make new Docker images, but that is not a problem.